assets/tests/components/nav/bottomNavMobile.test.tsx

```diff
@@ -59,4 +59,19 @@ describe("BottomNavMobile", () => {
     expect(screen.queryByTitle("Search")).toBeNull()
     expect(screen.queryByTitle("Search Map")).toBeInTheDocument()
   })
+
+  test("does not render nav item with title 'Detours' even if in test group", () => {
+    ;(getTestGroups as jest.Mock).mockReturnValue([TestGroups.DetourPanel])
```

Comment:
**suggestion:** you can use `jest.mocked` instead:
```suggestion
    jest.mocked(getTestGroups).mockReturnValue([TestGroups.DetourPanel])
```

Reply:
See also similar cases in `leftNav.test.tsx`.

Reply:
I thought about that, but I didn't do it in this PR because I was faced with the choice of either:
 - Make this test different from all the surrounding ones; OR
 - Explode this diff by fixing all of them.

Maybe this would be worth fixing as a follow-up? Or maybe as a pre-factor?

Reply:
I think having these tests be slightly different from the surrounding ones is fine.